src/actions/exportsActions.js

```diff
@@ -136,3 +137,25 @@ function exportStatistiquesCSV(dateDebut, dateFin, type, idType, conseillerIds,
     return { type: 'EXPORT_STATISTIQUES_CSV_FAILURE', error };
   }
 }
+
+// eslint-disable-next-line max-len
+function exportDonneesGestionnaires(filtreRole, filtreParNomGestionnaire, nomOrdre = 'nom', ordre = 1) {
+  return async dispatch => {
+    dispatch(request());
+    // eslint-disable-next-line max-len
+    await exportsService.getExportDonneesGestionnaires(filtreRole, filtreParNomGestionnaire, nomOrdre, ordre)
+    .then(exportGestionnairesFileBlob => dispatch(success(exportGestionnairesFileBlob)))
+    .catch(exportGestionnairesFileError => dispatch(failure(exportGestionnairesFileError)));
+  };
+
+  function request() {
+    return { type: 'EXPORT_GESTIONNAIRES_REQUEST' };
+  }
+  function success(exportGestionnairesFileBlob) {
+    const nameFile = `export-gestionnaires_${filtreRole.toLowerCase()}`;
+    return { type: 'EXPORT_GESTIONNAIRES_SUCCESS', exportGestionnairesFileBlob, nameFile };
+  }
+  function failure(exportGestionnairesFileError) {
+    return { type: 'EXPORT_GESTIONNAIRES_FAILURE', exportGestionnairesFileError };
+  }
```

Comment:
Pas utilisé dans le reducer
```suggestion
  }
  function failure() {
    return { type: 'EXPORT_GESTIONNAIRES_FAILURE', };
  }
```

Reply:
ou alors il fait passer error ?

Reply:
mis, puis retiré, puis remis.